lib/MessageDecoratorTrait.php

```diff
@@ -17,15 +17,6 @@
  */
 trait MessageDecoratorTrait
 {
-    /**
-     * The inner request object.
-     *
-     * All method calls will be forwarded here.
-     *
-     * @var MessageInterface
-     */
-    protected $inner;
```

Comment:
I took this out, and added it to the Request and Response decorators. In those places, I can declare it is `RequestInterface` and/or `ResponseInterface` and then phpstan can work out what is going on and be made happy.